pkg/linter/linter.go

```diff
@@ -206,7 +206,12 @@ func (l Linter) Lint(ctx context.Context) (report.Report, error) {
 
 	aggregate.Violations = append(aggregate.Violations, goReport.Violations...)
 
-	regoReport, err := l.lintWithRegoRules(ctx, input)
+	aggregates, err := l.collectAggregates(ctx, input)
```

Comment:
I think guarding this with a:

```go
	if len(input.Modules) > 1 {
		// ...
	}
```
Would be enough to avoid having these rules called when only a single file is to be linted.